### DISCUSSION TITLE
Group and sort bookmarks

### DISCUSSION THREAD

**Discussion by DimitrisAthanasiadis** [dailydotdev/daily#520] [Feature Requests]
It would be very convenient to group and sort our bookmarks. For example, grouping/filtering by tag would speed up my bookmark search or, even, allow me to read only the topic I am interested in at the moment. This could be extended to more filtering and sorting options.

**Comment by idoshamun** [comment]
Hi @DimitrisAthanasiadis,

Thanks for the suggestion. We have a feature we call "Bookmark lists" on our backlog. So stay tuned

**Comment by DimitrisAthanasiadis** [comment]
Hi @idoshamun ,

Sorry it slipped my attention. Thank you for your quick response.

**Reply by idoshamun** [reply]
No worries 😊

**Comment by Hoss3770** [comment]
Maybe you can allow users to create their own lists. Sth like youtube playlists. When saving the user can tick checkboxes to save to the lists of his choice.

**Comment by CatinhoCR** [comment]
Just my POV.
Given the nature of how this _amazing_ extension works, this should be a high priority in the project's roadmap.

As a full-time developer I open more browser tabs in a day than I could count & more often than not Im looking for something specific to do my job so I can't spare 3-8 mins at the time to read the cool article(s) I got on my feed... 
While bookmarking them is usually good enough, after a few bookmarks a day it starts getting clumsy and out of control.. Collection/Categories would make this extension sooo much better than it already is.

Sry for the lengthy comment.

**Reply by idoshamun** [reply]
Thanks for sharing your perspective :)

**Comment by dineshsrd** [comment]
- Sharing the bookmarks folder, either all the bookmarks or specific few

If you plan to bring paid plans in future this could be a paid feature.

**Comment by dlastnoel** [comment]
Are there any updates regarding this feature request?

**Reply by idoshamun** [reply]
Not at the moment

**Comment by xeniode** [comment]
Any updates? 

Sorting by squads and source domain would be great too.

**Reply by idoshamun** [reply]
Still not prioritized on our end 😿

**Comment by alejoevilches** [comment]
Hello devs!

Any updates on this? It would be a great feature to organize our bookmarks.

**Reply by idoshamun** [reply]
There's no update yet

**Comment by dreamy32** [comment]
I am looking forward this feature!

I often bookmark post on daily dev, but I have no way to organize them. On instagram and facebook, I created bookmark collections such as Front-End, CSS, Product Management and more.

This allow me to keep track of my bookmarks for reading later.

This would be a game-changer.

**Comment by Katelin-Frederick** [comment]
Looking forward to this feature as well!

I have quite a few bookmarks so this would be very helpful. I was just about to put in my own feature request for this then realized this one was already created.

Any updates on this?

**Reply by dreamy32** [reply]
They just wont answer. Take this as a no. :(

**Reply by idoshamun** [reply]
As you can see, I replied above several times. We're considering to release some more advanced capabilities as paid features, but that's still experimental

**Reply by dreamy32** [reply]
> As you can see, I replied above several times. We're considering to release some more advanced capabilities as paid features, but that's still experimental

Thanks for the update Ido!

I am happy to hear this. However, I can't hide I am disapointed about it being a paid feature. 

Wishing you well! 💪

**Reply by idoshamun** [reply]
Thanks a lot! I understand your disappointment but we need to make money as well, don't we?
